### ISSUE TITLE
SE block实现在arm上跑有错误

### ISSUE THREAD

**Issue by brightown** [Tencent/ncnn#1716]
se block实现通过ncnnoptimize后将innerproduct 和sigmoid合并，param文件类似：
InnerProduct             247                      1 1 246 248 0=64 1=1 2=256 9=4
ncnn的arm实现在https://github.com/Tencent/ncnn/blob/3a34b99fe6a5a2ef0119b412b420f3c8ff3f14dd/src/layer/arm/innerproduct_arm.cpp#L535
缺少activation_type == 4时sigmoid的处理

neon不熟，暂时加入：
         else if (activation_type ==4){
             sum0 = static_cast<float>(1.f / (1.f + exp(-sum0)));   
             sum1 = static_cast<float>(1.f / (1.f + exp(-sum1)));   
             sum2 = static_cast<float>(1.f / (1.f + exp(-sum2)));   
             sum3 = static_cast<float>(1.f / (1.f + exp(-sum3)));
         }
可以解决

请老大们处理～

**Comment by nihui** [comment]
修好了，感谢反馈

**Comment by ltss1988** [comment]
se class如下
class SeModule(nn.Module):
    def __init__(self, in_size, reduction=4):
        super(SeModule, self).__init__()
        self.pool = nn.AdaptiveAvgPool2d(1)
        self.se = nn.Sequential(
            nn.Conv2d(in_size, in_size // reduction, kernel_size=1, stride=1, padding=0, bias=False),
            nn.BatchNorm2d(in_size // reduction),
            nn.ReLU(inplace=True),
            nn.Conv2d(in_size // reduction, in_size, kernel_size=1, stride=1, padding=0, bias=False),
            nn.BatchNorm2d(in_size),
            nn.Sigmoid()
        )
    def forward(self, x):
        return x * self.se(self.pool(x)) 
在转onnx没有问题，Linux转ncnn也没有问题，安卓模拟器也没有问题，在安卓手机不出结果，对比输出是在 nn.Conv2d(in_size, in_size // reduction, kernel_size=1, stride=1, padding=0, bias=False)这层开始不对，请问这是什么原因

**Comment by ltss1988** [comment]
@nihui

**Comment by nihui** [comment]
> se class如下
> class SeModule(nn.Module):
> def **init**(self, in_size, reduction=4):
> super(SeModule, self).**init**()
> self.pool = nn.AdaptiveAvgPool2d(1)
> self.se = nn.Sequential(
> nn.Conv2d(in_size, in_size // reduction, kernel_size=1, stride=1, padding=0, bias=False),
> nn.BatchNorm2d(in_size // reduction),
> nn.ReLU(inplace=True),
> nn.Conv2d(in_size // reduction, in_size, kernel_size=1, stride=1, padding=0, bias=False),
> nn.BatchNorm2d(in_size),
> nn.Sigmoid()
> )
> def forward(self, x):
> return x * self.se(self.pool(x))
> 在转onnx没有问题，Linux转ncnn也没有问题，安卓模拟器也没有问题，在安卓手机不出结果，对比输出是在 nn.Conv2d(in_size, in_size // reduction, kernel_size=1, stride=1, padding=0, bias=False)这层开始不对，请问这是什么原因

请提供下 param 文件

**Comment by ltss1988** [comment]
SE 模块代码 param：
Convolution      402                      1 1 401 402 0=40 1=1 11=1 2=1 12=1 3=1 13=1 4=0 14=0 15=0 16=0 5=0 6=3840
BatchNorm        403                      1 1 402 403 0=40
Split            splitncnn_2              1 2 403 403_splitncnn_0 403_splitncnn_1
Pooling          404                      1 1 403_splitncnn_1 404 0=1 4=1
Convolution      405                      1 1 404 405 0=10 1=1 11=1 2=1 12=1 3=1 13=1 4=0 14=0 15=0 16=0 5=0 6=400
BatchNorm        406                      1 1 405 406 0=10
ReLU             407                      1 1 406 407
Convolution      408                      1 1 407 408 0=40 1=1 11=1 2=1 12=1 3=1 13=1 4=0 14=0 15=0 16=0 5=0 6=400
BatchNorm        409                      1 1 408 409 0=40
Sigmoid          410                      1 1 409 410
BinaryOp         411                      2 1 403_splitncnn_0 410 411 0=2
Split            splitncnn_3              1 2 411 411_splitncnn_0 411_splitncnn_1

**Comment by nihui** [comment]
针对onnx模型转换的各种问题，推荐使用最新的pnnx工具转换到ncnn
In view of various problems in onnx model conversion, it is recommended to use the latest pnnx tool to convert your model to ncnn

```shell
pip install pnnx
pnnx model.onnx inputshape=[1,3,224,224]
```

详细参考文档
Detailed reference documentation
https://github.com/pnnx/pnnx
https://github.com/Tencent/ncnn/wiki/use-ncnn-with-pytorch-or-onnx#how-to-use-pnnx
